notion/README.md

```diff
@@ -0,0 +1,37 @@
+# Agent Check: Notion
+
+## Overview
+
+Notion is a Connected Workspace. It's where modern teams create and share docs, take notes, manage projects, and organize knowledge, all in one place. Install Notion's integration with Datadog to manage and monitor your workspace activity in [Datadog Cloud SIEM][1]. You can import your workspace's audit logs for real-time monitoring, alerting, and analysis. From there, you can detect and investigate potential security issues, suspicious behavior, and troubleshoot access with confidence and ease.
+
+## Setup
+
+Datadog setup
+
+Go to the Configuration tab and click Install Integration at the bottom.
+
+Click Connect Accounts to redirect to Settings & Members in Notion
+
+Login to Notion then navigate to Connections > Workspace Connections > +Add Connection > Datadog. 
+
+Notion will prompt you to follow a series of OAuth steps to authorize the integration with Datadog.
+
+Once connected, Notion will start sending near real-time data to Datadog.
```

Comment:
```suggestion
1. Open the Notion tile and click _Install Integration_.

2. Click _Connect Accounts_ to redirect to _Settings & Members_ in Notion.

3. Login to Notion, then navigate to _Connections_ > _Workspace Connections_ > _+Add Connection_ > _Datadog_. 

4. Notion prompts you to follow a series of OAuth steps to authorize the integration with Datadog.

Once connected, Notion starts sending near real-time data to Datadog.
```

1. let's also add a link to the tile in the first step
2. after you click Connect Accounts and redirect, is there anything else you do on that Settings & Members page?
3. only in the third step do you login to Notion... will the previous step redirect properly if you don't login beforehand?